pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -415,3 +417,13 @@ def teardown(self) -> None:
             self.lightning_module.cpu()
             # clean up memory
             torch.cuda.empty_cache()
+
+    @staticmethod
+    def _clean_logger(*args: Any) -> None:
```

Comment:
wasn't the argument that by moving the `_log_hyperparameters` we wouldn't need this?

Reply:
I thought so, but it still hanged.

Reply:
Reverted the change.